hudi-sync/hudi-sync-common/src/main/java/org/apache/hudi/sync/common/AbstractSyncHoodieClient.java

```diff
@@ -29,20 +29,25 @@
 
 import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.Path;
+import org.apache.hudi.common.util.ValidationUtils;
 import org.apache.log4j.LogManager;
 import org.apache.log4j.Logger;
 import org.apache.parquet.schema.MessageType;
 
+import java.io.Serializable;
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.Statement;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
 
 public abstract class AbstractSyncHoodieClient {
 
   private static final Logger LOG = LogManager.getLogger(AbstractSyncHoodieClient.class);
 
+  public static final TypeConverter DECIMAL_TYPE_CONVERTOR = new TypeConverter() {};
```

Comment:
rename to `TYPE_CONVERTOR`?

Reply:
IMO, The current default implementation is the default implementation of the decimal type convertor. And we would do a special implementation by extending `TypeConverter` in the specific scene. So, keeping `DECIMAL_TYPE_CONVERTOR` would be a fine choice in the case...

Reply:
@leesf Done. Thank you for your suggestions.